test.js

```diff
@@ -204,7 +204,7 @@ test('assign styles', t => {
 });
 
 test('assign other props', t => {
-	const el = <a href="video.mp4" id="a" download referrerPolicy="no-referrer">Download</a>;
```

Comment:
Gotcha. Are you sure that it doesn't `just work` as is? I don't think HTML attributes are case sensitive and SVG ones are already react-style. Perhaps we can see how React's own createElement handles them.